interpreter/text/parser.mly

```diff
@@ -572,6 +572,7 @@ offset :
 elemref :
   | LPAR REF_NULL RPAR { let at = at () in fun c -> Null @@ at }
   | LPAR REF_FUNC var RPAR { let at = at () in fun c -> Func ($3 c func) @@ at }
+  | var { let at = at () in fun c -> Func ($1 c func) @@ at }
```

Comment:
I would strongly prefer dropping this form, as it is an odd one out that doesn't really scale beyond what we currently have.

Reply:
That's true, but the other forms are quite a bit more verbose in the examples. Still, I'm fine dropping it.

Reply:
Okay, please see PR #84.